jsx/App/locale/LocaleConstants.jsx

```diff
@@ -237,3 +237,9 @@ export const latexLibraryText = {
   [ESPANOL] : "Formateado para paquetes LaTeX gb4e y gb4e-modified:",
   [FRANCAIS] : "Formaté pour les packages LaTeX gb4e et gb4e-modified:",
 };
+
+export const lingviewBrandingText = {
```

Comment:
Let's change the variable name to something like `footerText` so that it's more obvious where the text shows up.